lib/stats_by_route_date.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const afterAll = (opt) => {
+	let materialized = false
+	if (opt.statsByRouteIdAndDate === 'materialized-view') {
+		materialized = true
+	} else if (opt.statsByRouteIdAndDate !== 'view') {
+		throw new Error('invalid opt.statsByRouteIdAndDate, must be one of these: none, view, materialized-view.')
+	}
+	const createViewCmd = materialized
+		? `CREATE MATERIALIZED VIEW`
+		: `CREATE OR REPLACE VIEW`
```

Comment:
`CREATE OR REPLACE VIEW` looks like idempotency intended, while for `MATERIALIZED VIEW`,a preexisting view is not dropped(?)

Reply:
True, in order to stay simple, `gtfs-via-postgres` is *not* intended for idempotency. I just followed the rest of the code base here. I will adapt all the occurences eventually.